.github/workflows/ci.yml

```diff
@@ -62,22 +65,32 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm install sequelize@${{ matrix.sequelize-version }}
-      - run: npm run build
-      - run: npm run test
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn add sequelize@${{ matrix.sequelize-version }} --ignore-engines
+      - run: yarn test
   release:
     name: Release
     runs-on: ubuntu-latest
-    needs: [test-sqlite, test-postgres, test-mysql]
-    if: github.event_name == 'push' && github.ref == 'refs/heads/release'
+    needs:
+      [
+        test-sqlite,
+        test-postgres,
+        test-mysql
+      ]
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/v6' || github.ref == 'refs/heads/v7')
```

Comment:
we need to react to pushed to the `release` branch here.

Reply:
See reasoning in the comment about the branches above